tests/e2e/restart_test.go

```diff
@@ -84,7 +84,7 @@ func (s *RestartTestData) TestVolumeDetached(c *C) {
 	c.Assert(err, IsNil, Commentf(out))
 
 	// 4. Verify detached status. Volume should be detached (within the timeout)
-	status = verification.VerifyDetachedStatus(s.volumeName, s.config.DockerHosts[1], s.config.EsxHost)
+	status = verification.PollDetachedStatus(s.volumeName, s.config.DockerHosts[1], s.config.EsxHost)
```

Comment:
Yes. That's the intention. Since there is a clear distinction between to case, (to retry and not to retry), hence the two utils. Some of the test codes is bound to change.